### DISCUSSION TITLE
Move to another server

### DISCUSSION THREAD

**Discussion by VagueGit** [Mimetis/Dotmim.Sync#1115] [Q&A]
We have DMS syncing SQLite and SQL Server via https. We are considering moving our sql server databases to another server.

I'm hoping that using DMS we can maybe avoid the pain of trying to move all those databases (and their data). We could create new databases on the new server and sync the data on the clients to the new server.

I've looked at [Dotmim.Sync/Samples/AlreadyExistingDatabases](https://github.com/Mimetis/Dotmim.Sync/tree/master/Samples/AlreadyExistingDatabases)

From what I see, if we Deprovison the client, then await agent.SynchronizeAsync, then await agent.LocalOrchestrator.UpdateUntrackedRowsAsync(); that should get us syncing on the new server.

Thoughts?

**Comment by Mimetis** [comment]
It seems to be a good idea, and worth a try.
But, why are you not trying to make a backup restore of the server database instead ?

**Comment by VagueGit** [comment]
Backup restore gets a bit tedious when there are 700+ databases to move. Using DMS allows our customers to choose when they move.
